File: Week01/Decimal_to_Binary/Built_in.cs
using static System.Console;
using static System.Convert;


class Built_in
{
    static string decToBinary(int n)
    {
        return ToString(n, 2);      // Convert.ToString() method
    }

    static void Main()
    {
        int n = 12;
        WriteLine(decToBinary(n));
    }
}

File: Week01/Decimal_to_Binary/Built_in.cpp
#include<iostream>
#include<cmath>
#include<bitset>
#include<string>

using namespace std;


string decToBinary(int n) 
{
    int len = (int)(log2(n));

    return bitset<64>(n).to_string().substr(64 - len - 1);
}

int main() {
    int n = 12;
    cout << decToBinary(n);
    return 0;
}